data/src/sensors.cpp

```diff
@@ -60,57 +24,85 @@ void Sensors::calibrate() {
 bool Sensors::update() {
     update_status();
 
-    // Update raw sensors values
-    raw_data.adxl = adxl1.get_data();
-    raw_data.adxl_aux = adxl2.get_data();
-
-    raw_data.bmi_accel = bmi1.get_accel_data();
-    raw_data.bmi_gyro = bmi1.get_gyro_data();
-    raw_data.bmi_aux_accel = bmi2.get_accel_data();
-    raw_data.bmi_aux_gyro = bmi2.get_gyro_data();
-
-    raw_data.bmp = bmp1.get_sensor_data();
-    raw_data.bmp_aux = bmp2.get_sensor_data();
+    // Update raw sensors values and write them to the GOAT
+    auto temp_adxl(adxl1.get_data());
+    Data::get_instance().write(Data::NAV_SENSOR_ADXL1_DATA, &temp_adxl);
+    temp_adxl = adxl2.get_data();
+    Data::get_instance().write(Data::NAV_SENSOR_ADXL2_DATA, &temp_adxl);
+    auto temp_bmi(bmi1.get_accel_data());
+    Data::get_instance().write(Data::NAV_SENSOR_BMI1_ACCEL_DATA, &temp_bmi);
+    temp_bmi = bmi1.get_gyro_data();
+    Data::get_instance().write(Data::NAV_SENSOR_BMI1_GYRO_DATA, &temp_bmi);
+    temp_bmi = bmi2.get_accel_data();
+    Data::get_instance().write(Data::NAV_SENSOR_BMI2_ACCEL_DATA, &temp_bmi);
+    temp_bmi = bmi2.get_gyro_data();
+    Data::get_instance().write(Data::NAV_SENSOR_BMI2_GYRO_DATA, &temp_bmi);
+    auto temp_bmp(bmp1.get_sensor_data());
+    Data::get_instance().write(Data::NAV_SENSOR_BMP1_DATA, &temp_bmp);
+    temp_bmp = bmp2.get_sensor_data();
+    Data::get_instance().write(Data::NAV_SENSOR_BMP2_DATA, &temp_bmp);
```

Comment:
do we need to get the instance every time or can we store it in a temp var write in the function ?